performance_test/scripts/fio-cmp.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kbench-pvc-1
+spec:
+  volumeMode: Filesystem
+  #volumeMode: Block
+  #storageClassName: longhorn  # replace with your storage class
+  storageClassName: local-path
```

Comment:
For comparison, we probably can make the value of storage class configurable later, so it will not just be limited to local path and longhorn, or even only 2 volumes. 

We can do this when we add other solutions to compare in the future.